### ISSUE TITLE
Latex formula is not be rendered in pdf

### ISSUE THREAD

**Issue by Tseing** [kernoeb/docker-docsify-pdf#11]
I tried to move [docsify-latex](https://github.com/scruel/docsify-latex) plugin and Mathjax.js into `resources/js`.  This is my `js` directory:

```text
js
├─docsify-latex.min.js
├─docsify-puml.min.js
├─docsify.js
├─icons.js
├─MathJax.js
├─prism-bash.min.js
├─prism-json.min
└─prism-python.min.js
```

And I generate pdf files with GitHub action. This is my GitHub action job:

```yaml
- name: Generate PDF
  run: |
    docker run --rm \
    --cap-add=SYS_ADMIN \
    --user "$(id -u):$(id -g)" \
    -v "$(pwd)/docs:/home/node/docs:ro" \
    -v "$(pwd)/pdf:/home/node/pdf:rw" \
    -v "$(pwd)/resources/covers/cover.pdf:/home/node/resources/cover.pdf:ro" \
    -v "$(pwd)/resources/js/:/home/node/resources/js/:rw" \
    -e "PDF_OUTPUT_NAME=$GENERATED_FILE" \
    ${{ env.IMAGE_TAG }}
```

But all math formulas are not rendered in pdf. Could you please help me solve this problem?

![图片](https://github.com/user-attachments/assets/398c278e-1326-4b60-94b4-a44086582cc2)

**Comment by kernoeb** [comment]
It seems that the plugin has requirements : 
<img width="819" alt="image" src="https://github.com/user-attachments/assets/4b231559-dcd3-4f41-b3f9-7fb7431639e1">

<img width="615" alt="image" src="https://github.com/user-attachments/assets/28acad50-df27-4763-8fed-6ac1f18c96d5">

I'm currently trying to debug this, i'll keep you posted

**Comment by kernoeb** [comment]
<img width="146" alt="image" src="https://github.com/user-attachments/assets/938a15b9-eeb6-4674-a3c5-435cb1a9f968">

Okay, it's the order of files that is important
I added a `sort` on the list of JS resources, you can pull / update

Rename `docsify-latex.js` to make it the last in the directory 

<img width="238" alt="image" src="https://github.com/user-attachments/assets/6b9c5ad6-9623-4b44-9dba-4c7639ae9036">

**Comment by Tseing** [comment]
Thank you! I think it would be better to make some descriptions about the impact of JS files' names in README. I don't know whether there are some ways to set the order of JS importing rather than depending on names.

**Comment by kernoeb** [comment]
Yes good idea, another way is to create a bundled JS file of both files combined, but i'm going to update the README right now :)

**Comment by Tseing** [comment]
Sorry to bother u again. It seems there are still some problems of Latex rendering. In my generated pdf,

- formula font is not be loaded.
- line height of inline formula is abnormal.

I dont know how to debug this. Latex formula appear very frequently in markdown and docsify documentations. Maybe we should consider to create a test case : )

**Comment by kernoeb** [comment]
Do you know if there is a CSS file that should be imported or something ? I'm not very familiar with it
If you wan to debug, you can just `npm run dev`, the chromium instance will be opened with the devtools
I'm at work, if you don't have any idea I'll check this tonight

**Comment by Tseing** [comment]
I got it. Formula rendering depend on not only single JS file but also some fonts files.

![图片](https://github.com/user-attachments/assets/558efc96-637a-4fd2-a789-407d1c640287)

I downloaded fonts from MathJax repo and move them into js folder. Then all is done : )

![图片](https://github.com/user-attachments/assets/9be134da-cd6b-4706-a437-7a4fa536891b)

![图片](https://github.com/user-attachments/assets/51620160-039f-4d68-83f7-15b1d9c2519a)

However I am not sure it is a good solution or not.

**Comment by kernoeb** [comment]
Actually it's not that bad, it seems logical, cool !

But maybe we should integrate Latex support natively in docker-docsify-pdf, with associated test, or at least give the procedure if it's not possible with the licence (i don't know if JS resources from docsify latex and Mathjax are MIT or anything like that)

**Comment by Tseing** [comment]
docsify-latex is released with MIT license and MathJax with Apache license. If we want to integrate  MathJax in docker-docsify-pdf, we have to keep original license.

**Comment by Tseing** [comment]
Okay, I will do it

**Comment by Tseing** [comment]
I found KaTex is MIT license. Or we can directly integrate it into docker-docsify-pdf?

**Comment by kernoeb** [comment]
Why not, nice

Maybe it just shouldn't be blocking, if the CDN is down or in a context with a firewall, it must not timeout, but I don't think that's really a problem, it's a very rare use case

Thanks ! Will test the PR tonight :)

**Comment by kernoeb** [comment]
Sorry, i couldn't test it yesterday, nice, thanks for your contribution !
